src/login/login.js

```diff
@@ -199,23 +207,22 @@ export default function loginFactory($container, config) {
 
             hide();
 
-            $inputToggle.on('click', function() {
-                if ($pwdInput.type === 'password') {
-                    show();
-                } else {
-                    hide(true);
+            $inputToggle.on('keyup', function (e) {
+                if (e.key === ' ') {
+                    togglePassword();
                 }
             });
 
-            $inputToggle.on('keyup', function(e) {
-                if (e.key === ' ') {
-                    if ($pwdInput.type === 'password') {
-                        show();
-                    } else {
-                        hide(true);
-                    }
+            $inputToggle.on('keypress', function (e) {
```

Comment:
Can  `keyup` event be used instead?

Reply:
keypress event is used in form submitting, in case of using keup submit will be done before
https://github.com/oat-sa/tao-core-ui-fe/blob/0b86959071db67fd6f9acb41538b26e225cd5091/src/login/login.js#L290